src/lib/holocene/banner/banner.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+  import IconButton from '$lib/holocene/icon-button.svelte';
+  import { translate } from '$lib/i18n/translate';
+  import { bannerClosed } from '$lib/stores/banner';
+
+  export let message: string;
+
+  $: show = message && !$bannerClosed;
+</script>
+
+{#if show}
+  <section class="banner" {...$$restProps}>
+    {message}
+    <IconButton
+      on:click={() => ($bannerClosed = true)}
+      data-testid="close-banner"
+      class="text-black-600 absolute top-0 right-0 mr-5 h-10 w-10"
+      icon="close"
+      label={translate('common.close')}
```

Comment:
Can we pass this label as a prop? I don't want Holocene to depend on i18next

Reply:
👍